mets_adapter/Makefile

```diff
@@ -0,0 +1,17 @@
+ROOT = $(shell git rev-parse --show-toplevel)
+include $(ROOT)/makefiles/functions.Makefile
+
+STACK_ROOT = mets_adapter
+
+SBT_APPS = mets_adapter
+
+SBT_DOCKER_LIBRARIES =
+SBT_NO_DOCKER_LIBRARIES =
+
+PYTHON_APPS =
+LAMBDAS	=
+
+TF_NAME = mets_adapter
```

Comment:
Is there going to be a mets_adapter terraform stack?

Reply:
I would expect so? This was just copied from another `Makefile`  so happy to remove this for now

Reply:
Nope, was just wondering where the infrastucture is going to live... It's fine to keep it if it's going to be useful